packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/routing/nelmio_cors.yaml

```diff
@@ -0,0 +1,10 @@
+nelmio_cors:
+    defaults:
+        origin_regex: true
+        allow_origin: ['%env(CORS_ALLOW_ORIGIN)%']
+        allow_methods: ['GET', 'OPTIONS', 'POST', 'PUT', 'PATCH', 'DELETE']
+        allow_headers: ['Content-Type', 'Authorization']
+        expose_headers: ['Link']
+        max_age: 3600
+    paths:
+        '^/': null
```

Comment:
The 2nd half should be the expected output.

Also, I think you can cut it down the to 3 lines. Only the 2nd level key is important.

```yaml
nelmio_cors:
    defaults:
        any_key: any_value
```

Btw, I got your wrong directory name. These will be always routes :) my bad.
Should be here: https://github.com/symplify/symplify/tree/main/packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal